channels/facebook/wphook.js

```diff
@@ -241,66 +249,71 @@ function processWorkplaceSecurityEvents(data) {
 
 function sendMessage(messageObj, session) {
 
+	var accessToken = WORKPLACE_PAGE_ACCESS_TOKEN
```

Comment:
I would keep this out as, if I understand correctly, it will never be used, right?
We will always fetch the token from session

Reply:
its actually still used for custom integrations. https://vonage.facebook.com/work/admin/?section=apps&ref=bookmarks&tab=custom

Reply:
any of the apps we work now will need it. but if the app gets distributed by FB, it wont need it

Reply:
OK